x/wasm/keeper/bench_test.go

```diff
@@ -1,6 +1,7 @@
 package keeper
 
 import (
+	sdk "github.com/line/lbm-sdk/types"
```

Comment:
How about relocating this import under 7L. Because I think it's better the import to include `github.com` locate behind golang library import files.